### DISCUSSION TITLE
discord button in Blish

### DISCUSSION THREAD

**Discussion by Taschenbuch** [blish-hud/Blish-HUD#738] [Blish HUD Core Ideas]
# Suggestion
It would be nice to have a blishud-discord button. This way users can easily find the right place to ask for help and are able to give feedback and report bugs.

# Requirements
- the discord link should be permanent. So no fancy discord-nitro-link that may get invalid after a while (if that is a thing)
- the discord link should be clickable so it directly opens discord in the browser. (see code example below)
- side note: maybe another button to the blish hud website could be useful too, though i think the directly-to-discord-button is much more important.
- the button should be visible and not too hidden in the user interface. possible button placements: 
   - on about page. Though that is a bit hidden. Maybe rename "About" page to "Help and Info" tab in this case
   - give the button its own "Help" tab.
   - add a "Help"-tab that is not a tab but on click opens the discord in browser. This tab should look like a button to prevent a confusing UX.
   - in the blish corner icon right click menu
   - do NOT add a "?"-button at the top right of the blish settings window or something similiar. That is too hidden.

![image](https://user-images.githubusercontent.com/43114787/188261370-f658fc4a-11d4-444b-935b-4a7a35a18a16.png)

example code
```
Process.Start(new ProcessStartInfo
{
    FileName = "https://discord.com/invite/FYKN3qh", // is that a permanent discord link? i guess so?
    UseShellExecute = true
});
```

**Comment by dlamkins** [comment]
We'd want to use the link: `https://link.blishhud.com/discord`

This link points to a proper invite link but still allows us to change it in the future if necessary.

I don't think we'd want to add it to the right-click menu on Blish HUD's own icon in-game as that menu is not easily discoverable.  I do wish to include it on the About page and perhaps in the tray area icon's right-click menu.  Those seem reasonable.

I agree that in any place it's shown, it should be clearly indicated that it is a button for Discord.

**Comment by dlamkins** [comment]
This button is now available in Blish HUD's about page.  😄
